src/main/java/com/example/demo/HW7.java

```diff
@@ -5,11 +5,14 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
+
 class HW7 implements BeanPostProcessor {
 
     @Override
+
     public Object postProcessBeforeInitialization(Object bean, String beanName) throws BeansException {
         System.out.println("Called postProcessBeforeInitialization() for: " + beanName);
```

Comment:
please try to use @Slf4j for a class and use log.info("") for logging. this is part of Lombok framework